### ISSUE TITLE
Support for Deno?

### ISSUE THREAD

**Issue by shah** [blakeembrey/javascript-stringify#36]
Hi there, this is a cool library but does not have built-in support for Deno. I've [forked the repo](https://github.com/shah/javascript-stringify) and made minor changes to get it to work on Deno. Any interest in rolling in Deno support to your master branch?

If you're interested, [Denoify](https://github.com/garronej/denoify) is a tool that could make it easier?

**Comment by blakeembrey** [comment]
Happy to accept a PR. I think all that’s needed is adding extensions to imports right?

**Comment by shah** [comment]
Yes @blakeembrey from the version I tested adding `*.ts` extensions is all that's required. I only need Deno support so that's all I need but I suspect you'd like your module to remain ready for NodeJS, the browser, or Deno so that's why I recommended Denoify. Also, I haven't been able to run tests in Deno yet (I haven't had a chance to research which libraries you're using) but that would be handy: Deno has built-in testing infrastructure and tooling.
